src/app/components/cds/inputs/TextArea.js

```diff
@@ -1,20 +1,37 @@
 // DESIGNS: https://www.figma.com/file/rnSPSHDgFncxjXsZQuEVKd/Design-System?type=design&node-id=3606-26274&mode=design&t=ZVq51pKdIKdWZicO-4
 import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import TextField from './TextField';
 
 const TextArea = React.forwardRef(({
-  autoGrow = true,
+  autoGrow,
+  maxHeight,
+  minHeight: minHeightProp,
   ...inputProps
 }, ref) => {
   const [height, setHeight] = useState('auto');
 
+
   const handleChange = (event) => {
-    if (autoGrow) {
-      setHeight(`${event.target.scrollHeight}px`);
+    const minHeight = minHeightProp || event.target.offsetHeight;
+    const newHeight = `${event.target.scrollHeight}px`;
+    if (autoGrow && event.target.scrollHeight >= minHeight && (!maxHeight || event.target.scrollHeight <= maxHeight)) {
+      setHeight(newHeight);
     }
```

Comment:
What happens if the user defines a custom `onChange`? I think we need to call it here:

```
const handleChange = (event) => {
  // [...] Current code here
  if (inputProps.onChange) {
    inputProps.onChange();
  }
}
```